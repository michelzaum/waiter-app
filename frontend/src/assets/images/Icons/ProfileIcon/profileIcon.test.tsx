import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import { ProfileIcon } from '.';

describe('ProfileIcon', () => {
  it('Should render the ProfileIcon', () => {
    //given
    const result = render(<ProfileIcon />);

    // then
    expect(result).toMatchSnapshot();
  });
});
