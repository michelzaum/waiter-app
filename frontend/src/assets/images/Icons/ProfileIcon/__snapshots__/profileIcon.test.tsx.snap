// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProfileIcon > Should render the ProfileIcon 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <svg
        fill="none"
        height="18"
        viewBox="0 0 20 18"
        width="20"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M12.1 6.40002C13.3 7.60002 13.3 9.50002 12.1 10.6C10.9 11.7 9.00002 11.8 7.90002 10.6C6.80002 9.40002 6.70002 7.50002 7.90002 6.40002C9.10002 5.30002 10.9 5.20002 12.1 6.40002M3.90002 17C5.50002 15.5 7.60002 14.5 10 14.5C12.3 14.5 14.5 15.4 16.1 17M2 14C1.4 12.8 1 11.4 1 10C1 5 5 1 10 1C15 1 19 5 19 10C19 11.4 18.6 12.8 18 14"
          stroke="currentColor"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="1.5"
        />
      </svg>
    </div>
  </body>,
  "container": <div>
    <svg
      fill="none"
      height="18"
      viewBox="0 0 20 18"
      width="20"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M12.1 6.40002C13.3 7.60002 13.3 9.50002 12.1 10.6C10.9 11.7 9.00002 11.8 7.90002 10.6C6.80002 9.40002 6.70002 7.50002 7.90002 6.40002C9.10002 5.30002 10.9 5.20002 12.1 6.40002M3.90002 17C5.50002 15.5 7.60002 14.5 10 14.5C12.3 14.5 14.5 15.4 16.1 17M2 14C1.4 12.8 1 11.4 1 10C1 5 5 1 10 1C15 1 19 5 19 10C19 11.4 18.6 12.8 18 14"
        stroke="currentColor"
        stroke-linecap="round"
        stroke-linejoin="round"
        stroke-width="1.5"
      />
    </svg>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
